mandatory/2-logic-error.js

```diff
@@ -1,16 +1,16 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
 
 function getStringLength(word) {
-  return "word".length();
+  return word.length();
```

Comment:
You do not need to use () when you call length(). It should be `word.length`, without a function call. 
Please check this: https://www.w3schools.com/jsref/jsref_length_string.asp